docs/src/styles/custom-css.css

```diff
@@ -139,11 +139,6 @@ starlight-menu-button svg {
   color: var(--sl-color-text);
 }
 
-.starlight-aside--note .starlight-aside__title,
-.starlight-aside--tip .starlight-aside__title {
-  color: white;
-}
```

Comment:
no you still need that, otherwise the title is blue: 
but you are right about the light mode, I always forgot to test it. 
you can use 
```css
  color: var(--sl-color-white);
```